src/frontends/onnx/frontend/src/op/blackmanwindow.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "op/blackmanwindow.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "utils/common.hpp"
+
+namespace ngraph {
```

Comment:
Some parts of code are marked as deprecated due to upcoming refactoring. That's why I get an deprecation warnings (raised as errors in CI).

To prevent deprecated warnings please add OPENVINO_SUPPRESS_DEPRECATED_START before this line...